src/cli/cli.cpp

```diff
@@ -8220,12 +8220,40 @@ template <> otError Interpreter::Process<Cmd("wakeupchannel")>(Arg aArgs[])
     return ProcessGetSet(aArgs, otLinkGetWakeupChannel, otLinkSetWakeupChannel);
 }
 
-#if OPENTHREAD_CONFIG_WAKEUP_END_DEVICE_ENABLE
 template <> otError Interpreter::Process<Cmd("wakeup")>(Arg aArgs[])
 {
     otError error = OT_ERROR_NONE;
     bool    enable;
 
+#if OPENTHREAD_CONFIG_WAKEUP_COORDINATOR_ENABLE
+    OT_UNUSED_VARIABLE(enable);
```

Comment:
Declaration of `enabled` can be moved to the `#if OPENTHREAD_CONFIG_WAKEUP_END_DEVICE_ENABLE` part.